runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -294,11 +294,28 @@ class INTERPRETER_CLASS
 	{
 		U_8 *profilingCursor = startProfilingRecord(REGISTER_ARGS, sizeof(J9Class*) + 2*sizeof(J9Method*));
 		if (NULL != profilingCursor) {
+#ifdef J9ARM
+			// ARMv8 does not support store multiple to unaligned addresses, and unfortunately GCC
+			// generates a store multiple for the normal version of these three assignments.
+			// To avoid the store multiple instructions, we can replace the assignments with
+			// inline assembly code.
+			asm volatile(
+				"str %[clazz],  [%[cursor], #0]\n\t"
+				"str %[method], [%[cursor], #4]\n\t"
+				"str %[target], [%[cursor], #8]\n\t"
+				:            // No outputs
+				:[cursor] "r" (profilingCursor),
+				 [clazz]  "r" (clazz),
+				 [method] "r" (callingMethod),
+				 [target] "r" (targetMethod)
+			 );
+#else
 			*(J9Class**)profilingCursor = clazz;
 			profilingCursor += sizeof(J9Class*);
 			*(J9Method**)profilingCursor = callingMethod;
 			profilingCursor += sizeof(J9Method*);
 			*(J9Method**)profilingCursor = targetMethod;
+#endif
```

Comment:
Should be commented.

Reply:
Just checking, you'd like a comment associated with the #else and #endif statements to connect them with the corresponding #ifdef? Or you'd like more comments on the individual inline assembly statements?

Reply:
Yeah, the "standard" here is a comment to match up the preprocessor directives (there are many examples in the file, I'm sure).